src/java/org/apache/cassandra/db/commitlog/CommitLog.java

```diff
@@ -129,6 +131,25 @@ private static CommitLog construct()
 
         // register metrics
         metrics.attach(executor, segmentManager);
+
+        String diskMode = " ";
+
+        switch (DatabaseDescriptor.getCommitLogDiskAccessMode())
+        {
+            case standard:
+                diskMode = "Standard (buffered)";
+                break;
+            case mmap:
+                diskMode = "MMap(memory mmaped)";
+                break;
+            case direct_io:
+                diskMode = "Direct (non-buffered)";
+                break;
+            default:
+                throw new IllegalArgumentException("Unknown commitlog disk access mode type: " + DatabaseDescriptor.getCommitLogDiskAccessMode());
```

Comment:
`   logger.info("Disk access mode for commitlog write is " + DatabaseDescriptor.getCommitLogDiskAccessMode() );`
And if we add some detailed descriptions in the cassandra.yaml as what I described  upper , I think there may have no needs for this part here. Just tell the user what disk mode we are using is enough.

Reply:
I think this is required and right place to show disk access mode and replay warn log when Direct-IO is used. Is this OK ?